test/unit/metrics/test_metricframe_aggregates.py

```diff
@@ -1040,3 +1040,105 @@ def test_2m_1sf_2cf():
         assert ratios_overall[mfn][('m', 'aa')] == min(ratio_overall_m_a)
         assert ratios_overall[mfn][('m', 'ba')] == pytest.approx(min(ratio_overall_m_b),
                                                                  rel=1e-10, abs=1e-16)
+
+
+class Test2m1sf1cfErrorHandlingCM:
+    # Metricframe containing a confusion matrix raises ValueErrors, since cells are non-scalar
+    def _prepare(self):
+        fns = {'confusion_matrix': skm.confusion_matrix}
```

Comment:
As mentioned above, there should also be a variant with `fns = { 'recall' : skm.recall_score, 'confusion_matrix': skm.confusion_matrix) }` so that we cover the case where some columns work, but others don't

Reply:
Can I alter the existing tests to have an extra metric, or do you prefer to both have tests with two metrics and tests with only the CM metric?

Reply:
I altered the tests to have an extra metric. Adding tests with only consumer_matrix and errors='coerce', always leads to empty series, so I did leave those tests out.